src/main/java/at/hannibal2/skyhanni/data/MayorAPI.kt

```diff
@@ -34,6 +34,14 @@ object MayorAPI {
     var candidates = mapOf<Int, MayorJson.Candidate>()
         private set
     var currentMayor: Mayor? = null
+        get() {
```

Comment:
please dont modify perks and activePerks on each current mayor access. instead use mayor api and config change event/real mayor update actions to determine the states